crates/bevy_asset/src/loader.rs

```diff
@@ -166,11 +169,41 @@ impl<'a> LoadContext<'a> {
         self.get_handle(AssetPath::new_ref(self.path(), Some(label)))
     }
 
+    /// Queues an asset contained within the primary asset to be loaded later as if
+    /// it were a separate file. This allows arbitrary assets to be loaded where
```

Comment:
Add a line break after the basic description; it helps the doc strings play nicer with IDE tooling.